test/general/header_inclusion_order_async_0.pass.cpp

```diff
@@ -0,0 +1,42 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+#if TEST_DPCPP_BACKEND_PRESENT
+#    include _PSTL_TEST_HEADER(async)
+#endif // TEST_DPCPP_BACKEND_PRESENT
+
+#include <vector>
+
+#include "support/utils.h"
+
+int
+main()
+{
+#if TEST_DPCPP_BACKEND_PRESENT
+
+    sycl::queue q = TestUtils::get_test_queue();
+
+    constexpr std::size_t n = 100;
+
+    using T = float;
+    using allocator = sycl::usm_allocator<T, sycl::usm::alloc::shared>;
+
+    allocator alloc(q);
+    std::vector<T, allocator> data(n, 1, alloc);
+
+    auto f = oneapi::dpl::experimental::reduce_async(TestUtils::make_device_policy(q), data.begin(), data.end());
+    f.wait();
+
+#endif // TEST_DPCPP_BACKEND_PRESENT
+
+    return TestUtils::done();
```

Comment:
I think we should pass `TEST_DPCPP_BACKEND_PRESENT` to indicate the test system that a test is skipped in case of DPCPP is not present.